src/pages/Index.tsx

```diff
@@ -1,10 +1,11 @@
 import { css } from '@emotion/react'
-import React, { useState } from 'react'
+import React, { useState, useCallback } from 'react'
 import logo from '/@/assets/logo.svg'
-import { Link } from 'react-router-dom'
+import { Link, useNavigate } from 'react-router-dom'
 import { useAppDispatch, useAppSelector } from '/@/store/hooks'
 import { increment } from '/@/store/slice/counter'
 
+// クエリがある場合は，招待リンクを踏んだパターンとする
```

Comment:
ここ現在の実装に対して言ってるのか TODO なのかわかりづらいから TODO 明示したほうがいい気がする
```suggestion
// TODO: クエリがある場合は，招待リンクを踏んだパターンとする
```